src/Tribe/Views/V2/Views/Latest_Past_View.php

```diff
@@ -12,7 +12,7 @@
 use Tribe\Events\Views\V2\Views\Traits\List_Behavior;
 use Tribe__Context;
 
-class Recent_Past_View extends View {
+class Latest_past_View extends View {
 
 	use List_Behavior;
 	/**
```

Comment:
* :no_entry_sign: Missing doc comment for class Latest_past_View
* :no_entry_sign: Class name is not valid; consider Latest_Past_View instead
* :no_entry_sign: Class name is not valid; consider Latest_Past_View instead

<sub>hash: e243b89aaa38f22040e7d54d28471c40</sub>